src/org/usfirst/frc4904/robot/commands/FlywheelSpinup.java

```diff
@@ -1,12 +1,29 @@
 package org.usfirst.frc4904.robot.commands;
 
 
+import java.util.function.Supplier;
 import org.usfirst.frc4904.robot.RobotMap;
 import org.usfirst.frc4904.robot.subsystems.Flywheel;
 import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
 
-public class FlywheelSpinup extends MotorConstant {
+public class FlywheelSpinup extends MotorConstant implements OverridableCommand {
+	protected static boolean flywheelSpeedOverride = false;
+
 	public FlywheelSpinup() {
 		super(RobotMap.Component.flywheel, Flywheel.SHOOTING_SPEED);
 	}
+
+	public FlywheelSpinup(Supplier<Double> speedSupplier) {
```

Comment:
coolio!
If it's only ever going to take a double that's never null, maybe consider a DoubleSupplier. Its marginally faster because it uses `double`s instead of `Double`s.